modules/date.js

```diff
@@ -0,0 +1,6 @@
+import { DateTime } from '../node_modules/luxon/build/es6/luxon.js';
+
+export default function currentDate() {
+  const now = DateTime.now();
+  return now.toLocaleString(DateTime.DATETIME_MED_WITH_SECONDS);
```

Comment:
- [ ] The luxon library doesn't return a date at the navbar area. It throws this error on the console: `Loading module from “http://127.0.0.1:5500/node_modules/luxon/build/es6/luxon.js” was blocked because of a disallowed MIME type (“text/html”).`
![chrome-capture](https://user-images.githubusercontent.com/33323077/148919912-6cf93bc2-ed5e-451e-b87d-9f8c39c7ccb0.jpg)